artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/StatQueue.java

```diff
@@ -139,7 +141,7 @@ public void requestFailed(ClientMessage reply) throws Exception {
       });
    }
 
-   private void printStats(String result) {
+   private void printStats(String result, int maxRows) {
```

Comment:
why do you need to pass this in? Is it not addressable at the object (class level var)

Reply:
To handle the case printStats will be called with a value different from the class level var, in the future.

Reply:
I agree atm its not needed but to be sure to not leave any broken path I'm going to remove the parameter `maxRows` from the signature `printStats(final ActionContext context, final String filter, int maxRows)` too.